docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -105,43 +105,53 @@ Map DAML primitives to Java types
-+-----------------------------------+---------------------------------------+
-
-.. _com.daml.ledger.javaapi.data.Unit: https://docs.daml.com/app-dev/bindings-java/javadocs/com/daml/ledger/javaapi/data/Unit.html
++--------------------------------+--------------------------------------------+------------------------+
+| DAML type                      | Java type                                  | Java Bindings          |
+|                                |                                            | Value Type             |
++================================+============================================+========================+
+| ``Int``                        | ``java.lang.Long``                         | `Int64`_               |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Decimal``                    | ``java.math.BigDecimal``                   | `Decimal`_             |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Text``                       | ``java.lang.String``                       | `Text`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Bool``                       | ``java.util.Boolean``                      | `Bool`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Party``                      | ``java.lang.String``                       | `Party`_               |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Date``                       | ``java.time.LocalDate``                    | `Date`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Time``                       | ``java.time.Instant``                      | `Timestamp`_           |
++--------------------------------+--------------------------------------------+------------------------+
+| ``List`` or ``[]``             | ``java.util.List``                         | `DamlList`_            |
++--------------------------------+--------------------------------------------+------------------------+
+| ``TextMap``                    | ``java.util.Map``                          | `TextMap`_             |
+|                                | Restricted to using ``String`` keys.       |                        |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Optional``                   | ``java.util.Optional``                     | `DamlOptional`_        |
++--------------------------------+--------------------------------------------+------------------------+
+| ``()`` (Unit)                  | **None** since the Java language doesn’t   | `Unit`_                |
```

Comment:
I couldn't quite follow this sentence - the "None" at the start confused me. Maybe "There is no equivalent Java type", then "Since..." instead?